playbooks/roles/ipa-identity-provider/README.md

```diff
@@ -0,0 +1,80 @@
+IPA identity provider configuration and test
+============================================
+
+This ansible role will install, configure and test IPA identity provider with an open source identity management system FreeIPA on OCP 4.x cluster. To use this role it is required to set the environment variables viz. IPA_ADMIN_PASSWORD, IPA_DM_PASSWORD and IPA_USER_PASSWORD as mentioned below on a control host(on a host where you are running this playbook).
+you can set the role variables with inputs instead of defaults.
+
+
+Requirements
+------------
+
+ - Running OCP 4.x cluster.
+
+
+Role Variables
+--------------
+
+| Variable                          | Required | Default                                                |Comments                                                                                                                                                                                                                                                                          |
+|-----------------------------------|----------|--------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ipa_identity_provider             | yes      | false                                                  | Flag to enable the execution for ipa-identity-provider playbook                                                                                                                                                                                                                  |
```

Comment:
Since default is given why is it Required? Variables are Required only when no default is given and is used somewhere when the flag is enabled.

Reply:
This flag is mentioned required and given its default as `false` because the role will be enabled only when this flag is set to `true` and passed as an external argument by the user.